src/main/java/net/mcreator/element/types/Tag.java

```diff
@@ -55,6 +57,9 @@ public String tagType() {
 		if (type.equals("Biomes"))
 			return "worldgen/biome";
 
+		if (type.equals("Damage Types"))

```

Comment:
I think we should not capitalize type

![image](https://github.com/MCreator/MCreator/assets/16374228/03eddae3-2f30-4c92-9040-43660e74a5f8)